File: Andromeda.AvaloniaApp/Helper/InstallationInfos.cs
using static GogApi.DotNet.FSharp.GalaxyApi;

namespace Andromeda.AvaloniaApp.Helpers
{
    public class InstallationInfos
    {
        public InstallerInfo InstallerInfo { get; set; }
        public string GameTitle { get; set; }

        public InstallationInfos(string gameTitle, InstallerInfo installerInfo)
        {
            this.GameTitle = gameTitle;
            this.InstallerInfo = installerInfo;
        }
    }
}

File: src/Andromeda/AvaloniaApp.FSharp/Helpers/InstallationInfos.fs
namespace Andromeda.AvaloniaApp.FSharp.Helpers

open GogApi.DotNet.FSharp.GalaxyApi

type InstallationInfos (gameTitle, installerInfo) =
    member val InstallerInfo: InstallerInfo = installerInfo with get, set
    member val GameTitle: string = gameTitle with get, set
